test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties

```diff
@@ -79,7 +79,7 @@ fr_FR=\u20AC
 fr_LU=\u20AC
 hi_IN=\u0930\u0942
 hr=\u00A4
-hr_HR=Kn
+hr_HR=EUR
```

Comment:
Since this is a table for currency symbols, it has to be EURO SIGN ('\u20AC')

Reply:
Made those changes, thank you